### DISCUSSION TITLE
discussion: remove exclusionary language

### DISCUSSION THREAD

**Discussion by netoleal** [typestack/class-validator#1797] [Feature Ideas]
### Description

There are a bunch of terms using words that, some time ago were not so relevant, but today we must to talk about it.
Expressions like "whitelisted" should be replaced for more appropriated alternatives like "allowedList" or something similar.

In my current company we are moving toward this direction and trying to clear out our codebase and stop using terms like these. 

That being said, I would love to continue using class-validator but without having to use these old unappropriated expressions.

### Proposed solution

A refactor Removing Exclusionary Language

Some material about it: [https://thenewstack.io/words-matter-finally-tech-looks-at-removing-exclusionary-language/](https://thenewstack.io/words-matter-finally-tech-looks-at-removing-exclusionary-language/)

**Comment by braaar** [comment]
I'm converting this to a discussion and closing other issues on the same topic since this is more fleshed out.
